Modules/timemodule.c

```diff
@@ -1343,6 +1347,26 @@ _PyTime_GetThreadTimeWithInfo(_PyTime_t *tp, _Py_clock_info_t *info)
     *tp = t;
     return 0;
 }
+#elif defined(_AIX)
+#define HAVE_THREAD_TIME
+static int
+_PyTime_GetThreadTimeWithInfo(_PyTime_t *tp, _Py_clock_info_t *info)
+{
+    thread_cputime_t tc;
```

Comment:
Please add a comment explaining why thread_cputime() is preferred over standard clock_gettime(CLOCK_THREAD_CPUTIME_ID). Add "bpo-40192: " prefix, to add a reference to the issue.